sdk/digitaltwins/digitaltwins/samples/dtdl/digitalTwins/buildingTwin.ts

```diff
@@ -0,0 +1,7 @@
+﻿exports.buildingTwin = {
```

Comment:
this feels weird, to use exports directly. For TS files we prefer using ESM syntax (e.g. export default)

Reply:
These are exactly the same DTDLs what have been used in .NET SDK. If it is not a blocker I would go with them for the parity.

---
In reply to: [447257147](https://github.com/Azure/azure-sdk-for-js/pull/9787#discussion_r447257147) [](ancestors = 447257147)